### PR TITLE
update NEWS.md

### PR DESCRIPTION
**Pull request by maelle**
Should the current version number be edited in DESCRIPTION?

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
+# devel
+
+## New features
+
+* The `rhub_check_list` class gained a `get_group()` and a `get_check()` methods returning a `rhub_check` object from the list, by group_id or check_id.
```
Comment (gaborcsardi) [inline NEWS.md]: I think the classes won't be exported? In this case we should not write news about them.

File: NEWS.md [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+
+* The `rhub_check` class gained a `browse()` method replacing the `web()` method for opening the URLs corresponding to a `rhub_check` object.
+
+* In `list_package_checks()` and `list_my_checks()` the `howmany` arguments now refers to checks submitted simultaneously, so if you submitted via `check_for_cran()` although it'd have created 3 builds, it'd count for one check as far as `howmany` is concerned, due to grouping in the API.
```
Comment (gaborcsardi) [inline NEWS.md]: I think we can leave this out, not a big deal.

File: NEWS.md (hunk 1 above)
Comment (maelle) [reply NEWS.md]: I disagree, the new methods are an important update to what you can do to the output of list functions?

Comment (gaborcsardi) [reply NEWS.md]: Sure, I meant keeping this bullet, but omitting the class names somehow.

File: NEWS.md (hunk 2 above)
Comment (maelle) [reply NEWS.md]: I removed this paragraph, thanks.

File: NEWS.md (hunk 1 above)
Comment (gaborcsardi) [reply NEWS.md]: I.e. just write about the values returned by `check_*()` functions?

Comment (maelle) [reply NEWS.md]: Ah right will fix that!

Comment (maelle) [reply NEWS.md]: is the new phrasing better?

**Comment by gaborcsardi** [comment]
Thanks!

**Pull request merged**